life_expectancy/main.py

```diff
@@ -2,38 +2,41 @@
 Module that contains the main function to run the process.
 """
 
+import os
 import argparse
 
 import pandas as pd
 
 from life_expectancy.cleaning import clean_data
-from life_expectancy.constants import DATA_DIR, FILE_NAME, FILE_PATH
+from life_expectancy.constants import DATA_DIR
 from life_expectancy.file_handlers import load_data, save_data
 
 
 def main(
-    file_path: str, file_name: str, data_dir_path: str, country: str = "PT"
-) -> pd.DataFrame:
+    data_dir_path: str, country: str = "PT", file_path: str = os.path.join(DATA_DIR, "eu_life_expectancy_raw.tsv")
+)-> pd.DataFrame:
     """
     Loads and clean raw dataset. Saves the cleaned dataset.
 
-    :param file_path: The file path.
-    :param file_name: The name of the file to save.
+    :param file_path: The file path to load.
     :param data_dir_path: The path of data directory.
     :param country: The name of country.
     :returns: A pandas dataframe with the cleaned data.
     """
     dataset = load_data(file_path)
     clean_dataset = clean_data(dataset, country)
-    save_data(clean_dataset, file_name, data_dir_path)
+    save_data(clean_dataset, f"{country.lower()}_life_expectancy.csv", data_dir_path)
 
     return clean_dataset
 
 
 if __name__ == "__main__":  # pragma: no cover
     parser = argparse.ArgumentParser(description="Clean and save life expectancy data")
     parser.add_argument(
-        "--country", type="str", default="PT", help="The country code to clean data for"
+        "--country", type=str, default="PT", help="The country code to clean data for"
+    )
+    parser.add_argument(
+        "--file_path", type=str, default=os.path.join(DATA_DIR, "eu_life_expectancy_raw.tsv"), help="Specify data file path"
```

Comment:
Here you are defining the `os.path.join(DATA_DIR, "eu_life_expectancy_raw.tsv` function again. It's not goot to repeat yourself. Maybe create a global constant named DEFAULT_FILEPATH and use it throughout the file?

Reply:
You're right :) so if I need to change this path, i only need change once. :)